### PR TITLE
Add permits-os to all necessary Charts

### PR DESCRIPTION
**Pull request by luthermonson**
https://github.com/rancher/dashboard/issues/5137 https://github.com/rancher/rancher/issues/36401

### CODE REVIEW

**Comment by aiyengar2** [comment]
Followup to https://github.com/rancher/charts/pull/1779

File: release.yaml [hunk 1]
```diff
@@ -7,19 +7,30 @@ fleet-crd:
...
 - 1.2.100+up1.2.1
 - 2.1.1+up2.1.1-rc1
 - 2.1.1+up2.1.1-rc2
 rancher-backup-crd:
 - 1.2.100+up1.2.1
-- 2.1.1+up2.1.1-rc1
 - 2.1.1+up2.1.1-rc2
+harvester-cloud-provider:
+- 100.0.1+up0.1.8
+- 100.0.0+up0.1.8
```
Comment (aiyengar2) [inline release.yaml:29]: should be removed

File: release.yaml [hunk 2]
```diff
@@ -7,19 +7,30 @@ fleet-crd:
...
 - 1.1.300+up1.1.3
 - 1.2.300+up1.2.3
+- 100.0.2+up1.1.3
+- 100.1.2+up1.2.3
 rancher-alerting-drivers:
 - 1.0.301
+- 100.0.2
 rancher-backup:
 - 1.2.100+up1.2.1
 - 2.1.1+up2.1.1-rc1
```
Comment (aiyengar2) [inline release.yaml:22]: can you remove this too? Since we removed -crd below

File: release.yaml [hunk 3]
```diff
@@ -77,3 +93,18 @@ rancher-cis-benchmark-crd:
...
+rancher-cis-benchmark:
+- 2.0.3-rc1
+rancher-cis-benchmark-crd:
+- 2.0.3-rc1
+rancher-gatekeeper:
+- 100.0.1+up3.6.0
+rancher-gatekeeper-crd:
+- 100.0.1+up3.6.0
+system-upgrade-controller:
+- 100.0.2+up0.3.1
```
Comment (aiyengar2) [inline release.yaml:105]: can you remove this?

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

**Review by aiyengar2 — APPROVED** [review]
LGTM

**Comment by phillipsj** [comment]
LGTM

**Pull request merged**